alert-engine/src/main/scala/io/phdata/pulse/alertengine/AlertEngineMain.scala

```diff
@@ -44,13 +45,39 @@ object AlertEngineMain extends LazyLogging {
 
     val executorService = Executors.newSingleThreadScheduledExecutor()
 
+    val solrServer = new CloudSolrServer(parsedArgs.zkHost())
+
+    val mailNotificationService =
+      new MailNotificationService(parsedArgs.smtpServer(),
+        parsedArgs.smtpPort(),
+        parsedArgs.smtpUser(),
+        parsedArgs.smtpPassword,
+        parsedArgs.smtp_tls())
+
+    val slackNotificationService = new SlackNotificationService
+
+    val notificationFactory =
+      new NotificationServices(mailNotificationService, slackNotificationService)
+
+    val config = try {
+      AlertEngineConfigParser.getConfig(parsedArgs.conf())
+    } catch {
+      case e: Exception =>
+        logger.error("Error parsing configuration, exiting", e)
+        System.exit(1)
```

Comment:
I don't think this System.exit is necessary anymore. I put it there so a scheduled task could kill the application, but now arg parsing is part of the main class not the task so it should be removed.

Reply:
OK, it's been updated.

Reply:
Did you miss a push? The `System.exit` is still here